### ISSUE TITLE
Fresh Install is not bootable

### ISSUE THREAD

**Issue by bryceac** [voidlinux/void-installer#16]
I have used Void a bit and liked the distribution quite a bit.

However, when I decided to make a gnome 3 LiveCD, since I needed a touch-friendly Linux system and went about doing test run installation, the installer just exits, with no option asking if I want to reboot like the images on the website do, and when I reboot the machine, I cannot even boot because the install thinks I am running an EFI system, instead of the BIOS system that VirtualBox is set to by default.

The steps used to create the image are those specified in the mklive script README, with a package setup similar to the following address, which were notes I put down when I made a MATE LiveCD.

http://wiki.brycecampbell.me/index.php?title=Void_Linux

**Comment by Vaelatern** [comment]
Did you really use this repo? This repo is not ready for non-developer use.

**Comment by Vaelatern** [comment]
https://github.com/voidlinux/void-mklive might be where you meant to go

**Comment by bryceac** [comment]
I was going to say, the README for void-mklive says that it uses the void installer, but it does have an installer shell script of its own and clearly says that it contains the utilities, so I guess it should be moved.

I posted it here, because everything else about the image worked properly other than the install process and the shell script itself still seems to have everything that it did when I made a MATE LiveCD which was before the April images appeared online.

I'll probably close this and reopen over on the void-mklive repo.
